src/test/java/com/microsoft/jfr/generation/JfcToJavaTest.java

```diff
@@ -0,0 +1,49 @@
+package com.microsoft.jfr.generation;
+
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.testng.Assert.*;
+
+public class JfcToJavaTest {
+
+    @Test
+    void jfcToJavaMap() throws IOException {
```

Comment:
Maybe an @After that deletes the generated file? I'd do an @Before that does the same thing. If the file exists, delete it.